addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2614,6 +2614,41 @@ export class OdooEditor extends EventTarget {
      *
      */
     initElementForEdition(element = this.editable) {
+        // Detect if the editable base element contain orphan inline nodes.
+        // If so we transform the base element HTML to put those orphans
+        // inside `<p>` containers.
+        const orphanInlineChildNodes = [...element.childNodes].filter(
+            (n) => !isBlock(n) && (n.nodeType === Node.ELEMENT_NODE || n.textContent.trim() !== "")
+        );
+        if (orphanInlineChildNodes.length) {
+            let currentP;
+            const childNodes = [...element.childNodes];
+            const tempEl = document.createElement('temp-container');
+            const appendAndRenewCurrentP = () => {
+                if (currentP && currentP.childNodes.length > 0 && currentP.innerHTML.trim() !== '') {
+                    tempEl.append(currentP);
+                }
+                currentP = document.createElement('p');
+                currentP.style.marginBottom = '0';
+            };
+            appendAndRenewCurrentP();
+            for (const node of childNodes) {
+                if (node.nodeType === Node.ELEMENT_NODE && node.tagName === 'BR') {
+                    if (!currentP.childNodes.length) {
+                        currentP.append(node);
```

Comment:
This looks weird to me. Are you sure we want to append the BR to the P ? And why specifically if it's empty ?

Reply:
In order to match the Editor behaviors.

In the editor : 
```
<p>a[]b</p>
```
> Enter
> Enter
```
<p>a</p>
<p><br></p>
<p>[]b</p>
```